### ISSUE TITLE
x64 update

### ISSUE THREAD

**Issue by chcg** [mackwai/NPPHexEditor#2]
See https://github.com/chcg/NPP_HexEdit, just found your repo. I did:
- vs2013 and newer
- x64 compatibility
- CI build with appveyor
- no functional changes, just fixes for compiler warnings
- as sourcecontrol  at sourceforge is cvs it is not as easy as with svn to take over the history to github, I also added the older versions from the zips

Maybe we could team up and provide a updated version after checkback with jens lorenz, if he could be reached.

**Comment by mackwai** [comment]
Yes, let's team up.  It would be good to merge our changes.  I don't know if we'll have any luck reaching Jens.  I wasn't able to reach him when I was working on my version of the hex editor last year, which is partly why I posted my changes on Github instead of sourceforge.

Have you tried to install and run my version on your computer?  I've only tried it out on two or three computers, so it would be good to try it on another computer to get some more assurance that it's working.

**Comment by chcg** [comment]
- Seems jens is not available in github und n++ community :-(
- I haven't tried your build yet.
- Just had a short look at the changes. A big amount seems to be around the new feature for the alternative hexadecimal symbols. Which is not my major concern. 
- Unfortunately you didn't split the real bugfixes into individual commits to figure out the relevant changes dealing with bugs from the ones dealing with the new feature.
- my interests are in fixes some relevant bugs and get a working x64 version

**Comment by mackwai** [comment]
I can send patch files for some of the bug fixes I made, if you want them.  I mostly worked on the code in SVN on my PC.  I did a fair amount of code cleanup, too, so if I'm going to do any merging myself, I'd rather merge your changes into my source than vice versa.
